src/Particle/ParticleSet.cpp

```diff
@@ -43,10 +43,12 @@ template<> int ParticleSet::Walker_t::cuda_DataSize = 0;
 
 void add_p_timer(std::vector<NewTimer*>& timers)
 {
-  timers.push_back(new NewTimer("ParticleSet::makeMove",timer_level_fine)); //timer for MC, ratio etc
-  timers.push_back(new NewTimer("ParticleSet::makeMoveOnSphere",timer_level_fine)); //timer for the walker loop
+  timers.push_back(new NewTimer("ParticleSet::makeMove",timer_level_fine)); //timer for moves
+  timers.push_back(new NewTimer("ParticleSet::makeMoveOnSphere",timer_level_fine)); //timer for NLPP moves
+  timers.push_back(new NewTimer("ParticleSet::donePbyP",timer_level_fine)); //timer for donePbyP
   TimerManager.addTimer(timers[0]);
```

Comment:
Please explain the stack-like creation of the timers followed by fixed indexing [0], [1], [2]. I would prefer to avoid magic indices e.g. myTimers[1]